src/ui.rs

```diff
@@ -11,6 +11,20 @@ use hbb_common::{
     tokio::{self, time},
 };
 use sciter::Value;
+
```

Comment:
style problem, please use style like this "used std::{fs, path::...}", we need to keep the same style.